services/movie.py

```diff
@@ -0,0 +1,30 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> Movie:
+    queryset = Movie.objects.all()
+    if genres_ids is not None:
+        queryset = queryset.filter(genres__in=genres_ids)
+    if actors_ids is not None:
+        queryset = queryset.filter(actors__in=actors_ids)
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None

```

Comment:
Use Optional[list[int]] because None as default. Check in all places(with all types)